@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/headers/core.hpp

```diff
@@ -1,3 +1,11 @@
 #define STABILTOR_MODE_ALWAYSENABLED 2
 #define STABILTOR_MODE_JOYSTICKONLY 1
```

Comment:
Just realised, we seem to be saying `STABILTOR` rather than `STABILATOR`?

Reply:
Resolved